docs/app/file-library.md

```diff
@@ -1,137 +1,219 @@
 # File Library
 
-> Directus includes a full-featured Digital Asset Management (DAM) system for storing, organizing, browsing, and
-> transforming your various files and assets.
+> The File Library Module aggregates all files within the Directus Project into one consolidated library. It is a
+> full-featured Digital Asset Management (DAM) system for storing, organizing, browsing, and transforming your various
+> files and assets.
+
+![File Library](https://cdn.directus.io/docs/v9/app-guide/file-library/file-library-20220305A/file-library-20220305A.webp)
```

Comment:
Can we use a screenshot without a folder named `ok` ?

Reply:
lol yes we can boss!

Reply:
ok!